skelebot/systems/scaffolding/scaffolder.py

```diff
@@ -20,7 +22,22 @@ def __format_variables(self, text):
                 text = text.replace(key_str, value)
 
         return text
-    
+
+    def __format_config(self, config):
+        return eval(self.__format_variables(str(config)))
+
+    def __load_github_template(self, owner, repo, branch="main"):
+        url = GITHUB_RAW.format(owner=owner, repo=repo, branch=branch, filepath="template.yaml")
+        response = requests.get(url)
+        yaml_text = self.__format_variables(response.text)
```

Comment:
It seems a little strange that we format the whole template file, then later run any extra prompts defined there, and then format the config section, which means we may end up formatting that section twice. Is there a way to only run format once?

Reply:
maybe, but if anything in the yaml requires variables from the basic scaffolding prompts, it may be best to insert that asap. Especially if there is a prompt in the template that requires one of the variables.